streaming/src/test/scala/org/apache/spark/streaming/MasterFailureTest.scala

```diff
@@ -215,8 +216,8 @@ object MasterFailureTest extends Logging {
 
     while(!isLastOutputGenerated && !isTimedOut) {
       // Get the output buffer
-      val outputBuffer = ssc.graph.getOutputStreams().head.asInstanceOf[TestOutputStream[T]].output
-      def output = outputBuffer.flatMap(x => x)
+      val outputQueue = ssc.graph.getOutputStreams().head.asInstanceOf[TestOutputStream[T]].output
+      def output = outputQueue.asScala.flatMap(x => x)
```

Comment:
for sure, I just wanted to try and make the changes as small as possible. I'll update this.